apps/e2e/ag-grid-storybook/src/app/ag-grid/ag-grid-stories.module.ts

```diff
@@ -28,7 +28,6 @@ const routes: Routes = [{ path: '', component: AgGridStoriesComponent }];
     SkyThemeModule,
     PreviewWrapperModule,
   ],
-  providers: [SkyThemeService],
```

Comment:
The preview wrapper now ensures that the theme service is pulled in and it is the one provided at the root level.